### PR TITLE
Status message small tweaks

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Death message will appear when your mob dies (no longer appears when you simply ghost).

## Changelog
:cl:
tweak: Death pop up will only appear when your mob dies, not on ghosting
/:cl:

### CODE REVIEW

**Comment by Mat05usz** [comment]
Finally 😔

File: code/modules/tgui_panel/stat.dm [hunk 1]
```diff
@@ -79,6 +90,7 @@
 	if(!is_ready())
 		return
 	window.send_message("stat/deadPopup", list())
+	addtimer(CALLBACK(src, .proc/clear_dead_poup), 3 MINUTES)
```
Comment (utilbird) [inline code/modules/tgui_panel/stat.dm]: Is this needed? Timers are somewhat expensive so we should try to avoid them where we don't need them

Comment (Archanial) [reply code/modules/tgui_panel/stat.dm]: Unless you have other idea how to make it clear by itself it is needed.
If it's that big of an issue I can remove it

Comment (ike709) [inline code/modules/tgui_panel/stat.dm]:
```suggestion
	addtimer(CALLBACK(src, .proc/clear_dead_popup), 3 MINUTES)
```

**Comment by ike709** [comment]
Not sure if I'm a fan of the auto-clearing at the cost of a timer. It's not hard to clear manually.

File: code/modules/tgui_panel/stat.dm [hunk 2]
```diff
@@ -69,6 +69,17 @@
 	payload["title"] = title
 	payload["text"] = text
 	window.send_message("stat/antagPopup", payload)
+	addtimer(CALLBACK(src, .proc/clear_antagonist_popup), 3 MINUTES)
```
Comment (Archanial) [inline code/modules/tgui_panel/stat.dm]:
```suggestion
```

File: code/modules/tgui_panel/stat.dm [hunk 3]
```diff
@@ -79,13 +90,14 @@
 	if(!is_ready())
 		return
 	window.send_message("stat/deadPopup", list())
+	addtimer(CALLBACK(src, .proc/clear_dead_popup), 3 MINUTES)
```
Comment (Archanial) [inline code/modules/tgui_panel/stat.dm]:
```suggestion
```

**Review by ike709 — APPROVED** [review]

**Review by utilbird — APPROVED** [review]
🔥

**Pull request merged**